Project 2 (13.12)/script.js

```diff
@@ -0,0 +1,107 @@
+//zad 1
+// const reverseText = (string) => {
+//     let newString = "";
+//     for (let i = string.length - 1; i >= 0; i--) {
+//       newString += string[i];
+//     }
+//     return newString
+// };
+
+// const reverseText = (string) =>{
+//     return string.split("").reverse().join("");
+
+// }
+
+// reverseText("infoShare"); // => erahSofni
+// reverseText("iSA"); // => ASi
+// reverseText("Adam"); // => madA
+
+//zad 2
+
+// const splitString = (string) => {
+//     const newString = string.split(" ");
+//     let firstWord = newString[0];
+//     let secondWord = newString[1];
+//     let firstLetterA = firstWord[0];
+//     let firstLetterB = secondWord[0];
+//     return `${firstLetterB+firstWord.slice(1)} ${firstLetterA+secondWord.slice(1)}`;
+
+//      console.log(firstWord);
+//      console.log(secondWord);
+//      console.log(firstletterA);
+//      console.log(firstletterB);
+
+//    }
+
+//    console.log(splitString("Adam Kocinski"))
+
+// zad 3
+// let arr1 = [1,2,3];
+// let arr2 = [4,2,8];
+// function multiDiff (a,b) {
+//     const multi1 = a.reduce((a,b)=>a*b);
+//     const multi2 = arr2.reduce((a,b)=>a*b);
+//     const difference = multi1 - multi2;
+//     return Math.abs(difference);
+
+//     }
+// console.log(multiDiff(arr1, arr2))
+
+//zad 4
+
+// const vowels = ["a", "e", "i", "o", "u"];
+
+// function howManyVowels(string) {
+//   let counter = 0;
+//   for (let letter of string) {
+//     if (vowels.includes(letter)) {
+//       counter++;
+//     }
+//   }
+//   return counter;
+// }
+// console.log(howManyVowels("alibaba"));
+
+//zad 5
+
+// Napisz funkcję, która przyjmie dwa argumenty - number - "n" i tablicę z numberami. Zwróć nową tablicę z "n"
+// największych numberów z tablicy wyjściowej, np. funkcja (2, [5, 3, 1, 4, 2]) -> [5, 4]. Przyda się funkcja Array.sort().
+
+
+// function highestNumber(n, array) {
+//     let sortedArray = array.sort().reverse().slice(0,n)
```

Comment:
`return array.sort().reverse().slice(0,n)`

Reply:
racja wygląda ładniej , jeszcze pracuje nad tym kiedy uzywam return , a kiedy musze to zdefiniowac do zmiennej ale wydaje mi się , że już to w miare opanowałem, praktycznie wszystkie te zadania były robione na bierząco na zajęciach